content/docs/components-and-props.md

```diff
@@ -16,57 +16,59 @@ prev: rendering-elements.html
 next: state-and-lifecycle.html
 ---
 
-Components let you split the UI into independent, reusable pieces, and think about each piece in isolation. This page provides an introduction to the idea of components. You can find a [detailed component API reference here](/docs/react-component.html).
+Komponenter låter dig bryta upp användargränssnittet i oberoende, återanvändbara bitar och tänka på varje bit isolerat. Den här sidan ger dig en introduktion till tanken bakom komponenter. Du kan hitta ett [detaljerat API för komponenter här](/docs/react-component.html).
 
-Conceptually, components are like JavaScript functions. They accept arbitrary inputs (called "props") and return React elements describing what should appear on the screen.
+Konceptuellt är komponenter som JavaScript-funktioner. De accepterar godtyckliga inputs (kallade "props") och returnerar React-element som beskriver vad som ska visas på skärmen.
 
-## Function and Class Components {#function-and-class-components}
+## Funktion och klasskomponenter {#function-and-class-components}
 
-The simplest way to define a component is to write a JavaScript function:
+Det enklaste sättet att definiera en komponent är att skriva en JavaScript-funktion:
 
 ```js
 function Welcome(props) {
-  return <h1>Hello, {props.name}</h1>;
+  return <h1>Hej, {props.name}</h1>;
 }
 ```
 
-This function is a valid React component because it accepts a single "props" (which stands for properties) object argument with data and returns a React element. We call such components "function components" because they are literally JavaScript functions.
+Denna funktion är en giltig Reakt-komponent eftersom den accepterar en enda "props" (som står för properties) objektargument med data och returnerar ett React-element. Vi kallar sådana komponenter "funktionskomponenter" eftersom de bokstavligen är JavaScript-funktioner.
 
-You can also use an [ES6 class](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes) to define a component:
+Du kan även använda en [ES6-klass](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes) för att definiera en komponent:
 
 ```js
 class Welcome extends React.Component {
   render() {
-    return <h1>Hello, {this.props.name}</h1>;
+    return <h1>Hej, {this.props.name}</h1>;
   }
 }
 ```
 
-The above two components are equivalent from React's point of view.
+De två ovannämnda komponenterna är identiska ur Reacts synvinkel.
 
 Classes have some additional features that we will discuss in the [next sections](/docs/state-and-lifecycle.html). Until then, we will use function components for their conciseness.
 
-## Rendering a Component {#rendering-a-component}
+Klasser har några ytterligare funktioner, som vi kommer att diskutera i [nästa avsnitt](/docs/state-and-lifecycle.html). Fram till dess kommer vi att använda för funktionskomponenter för deras tydlighet.
```

Comment:
`next sections` are plural and therefore I think `senare avsnitt` would be a better fit here

Reply:
@rloqvist Fixing. 👍